### PR TITLE
Backup settings

### LINKED ISSUE #491: Implement backup settings page

### PR DESCRIPTION
**Pull request by briansteffens**
Closes #491

### CODE REVIEW

**Comment by eatonphil** [comment]
Screenshot?

**Comment by briansteffens** [comment]
screen shot 2016-11-07 at 4 16 57 pm

File: src/linodes/linode/backups/layouts/SettingsPage.js [hunk 1]
```diff
@@ -1,9 +1,182 @@
...
+    }
+  }
+
+  render() {
+    const { dispatch } = this.props;
+    const { window, day, errors } = this.state;
+    const linode = this.getLinode();
+
+    const cancelModal = linodeId => (
+      <CancelBackupsModal linodeId={linodeId} dispatch={dispatch} />
```
Comment (eatonphil) [inline src/linodes/linode/backups/layouts/SettingsPage.js]: Nitpick, could you pull the definition of this class above here?

File: src/linodes/linode/backups/layouts/SettingsPage.js [hunk 2]
```diff
@@ -1,9 +1,182 @@
...
+import { linodes } from '~/api';
+import { cancelBackup } from '~/api/backups';
+import { getLinode } from '~/linodes/linode/layouts/IndexPage';
+import { setSource } from '~/actions/source';
+import { showModal, hideModal } from '~/actions/modal';
+
+export class CancelBackupsModal extends Component {
+  constructor() {
+    super();
+    this.state = { loading: false };
```
Comment (eatonphil) [inline src/linodes/linode/backups/layouts/SettingsPage.js]: Why are you using this? Isn't the modal hidden once the button has been pressed?

**Pull request merged**